components/org.wso2.carbon.status.dashboard.core/src/gen/java/org/wso2/carbon/status/dashboard/core/model/ManagerOverView.java

```diff
@@ -0,0 +1,78 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+package org.wso2.carbon.status.dashboard.core.model;
+
+import org.wso2.carbon.status.dashboard.core.bean.ManagerClusterInfo;
+
+/**
+ * Class that gives an overview for manager node.
```

Comment:
This comments indicates that this class has a method that gives an overview for manager node. But this is a modal class, which means it acts as a data model rather than an operator. Please change the class comment to correctly indicates its purpose.